testdata/checkquorum.txt

```diff
@@ -0,0 +1,322 @@
+# tick 2 election timeouts, since the followers were active in the current
+# interval (see messages above).
+tick-election 1
+----
+ok
+
+tick-election 1
+----
+WARN 1 stepped down to follower since quorum is not active
+INFO 1 became follower at term 1
+
+stabilize
+----
+> 1 handling Ready
+  Ready MustSync=false:
+  Lead:0 State:StateFollower
+  Messages:
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+> 2 receiving messages
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->2 MsgHeartbeat Term:1 Log:0/0 Commit:11
+> 3 receiving messages
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+  1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+> 2 handling Ready
+  Ready MustSync=false:
+  Messages:
+  2->1 MsgAppResp Term:2 Log:0/0
+  2->1 MsgAppResp Term:2 Log:0/0
+  2->1 MsgAppResp Term:2 Log:0/0
+  2->1 MsgAppResp Term:2 Log:0/0
+  2->1 MsgAppResp Term:2 Log:0/0
+> 3 handling Ready
+  Ready MustSync=false:
+  Messages:
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+> 1 receiving messages
+  2->1 MsgAppResp Term:2 Log:0/0
+  INFO 1 [term: 1] received a MsgAppResp message with higher term from 2 [term: 2]
+  INFO 1 became follower at term 2
+  2->1 MsgAppResp Term:2 Log:0/0
+  2->1 MsgAppResp Term:2 Log:0/0
+  2->1 MsgAppResp Term:2 Log:0/0
+  2->1 MsgAppResp Term:2 Log:0/0
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  INFO 1 [term: 2] ignored a MsgHeartbeatResp message with lower term from 3 [term: 1]
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  INFO 1 [term: 2] ignored a MsgHeartbeatResp message with lower term from 3 [term: 1]
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  INFO 1 [term: 2] ignored a MsgHeartbeatResp message with lower term from 3 [term: 1]
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  INFO 1 [term: 2] ignored a MsgHeartbeatResp message with lower term from 3 [term: 1]
+  3->1 MsgHeartbeatResp Term:1 Log:0/0
+  INFO 1 [term: 2] ignored a MsgHeartbeatResp message with lower term from 3 [term: 1]
+> 1 handling Ready
+  Ready MustSync=true:
+  HardState Term:2 Commit:11
+
+# Other nodes can now successfully campaign. Note that we haven't ticked 3, so
+# it won't grant votes.
```

Comment:
You can highlight this by stabilizing 3 separately in the next step. That way, the salient bit won't be buried in a few pages of text.